src/voice-broadcast/components/molecules/VoiceBroadcastRecordingPip.tsx

```diff
@@ -31,11 +35,22 @@ interface VoiceBroadcastRecordingPipProps {
 export const VoiceBroadcastRecordingPip: React.FC<VoiceBroadcastRecordingPipProps> = ({ recording }) => {
     const {
         live,
-        sender,
+        recordingState,
         room,
+        sender,
         stopRecording,
+        toggleRecording,
     } = useVoiceBroadcastRecording(recording);
 
+    const toggleControl = recordingState === VoiceBroadcastInfoState.Paused
+        ? <VoiceBroadcastControl
+            className="mx_VoiceBroadcastControl-recording"
```

Comment:
nitpick: should be `_recording` instead of `-recording`

Reply:
With these BEM like notation from the [style guide](https://github.com/vector-im/element-web/blob/develop/code_style.md#stylesheets-pcss--postcss--plugins) shouldn't it even be `x_VoiceBroadcastControl--recording`?

Our style guide is still confusing me :D